assets/src/components/developers/RunKit.ts

```diff
@@ -88,18 +88,22 @@ async function handler({event, payload}) {
 
 async function handleMessageCreated(message) {
   const {body, customer_id, conversation_id} = message;
-  const isCustomerMessage = !!customer_id;
 
-  if (isCustomerMessage && body.toLowerCase().startsWith('test')) {
+    messageBody = body.toLowerCase()
+    let responseMessage = 'This is the default response'
+    
+    if(messageBody.includes('test')) {
+      responseMessage = "test successful!"
+    } else if (messageBody.includes('pricing')) {
+      responseMessage = "See our pricing page https://papercups.io/pricing"
+    } 
+    
     // See https://docs.papercups.io/api-endpoints#messages
     return papercups.messages.create({
-        body: 'Success!',
+        body: responseMessage,
```

Comment:
could clean this up by just moving this into a function, e.g. `getResponseMessage(body)` where `getResponseMessage` looks like
```js
function getResponseMessage(body) {
  const formatted = body.toLowerCase();

  if (formatted.includes('test')) {
    return 'Test successful!';
  } else if (formatted.includes('pricing')) {
    return 'Check out our pricing at [papercups.io/pricing](https://papercups.io/pricing)';
  } else {
    return `This is the default message. Received: ${body}`;
  }
}
```